### DISCUSSION TITLE
Welcome to Discussions about the Server part of Open Food Facts

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/openfoodfacts-server#4925] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by M123-dev** [comment]
One month in no answer, I think discussions are probably not quite so accepted
